test/com/serotonin/mango/util/AbstractStartStopDataPointsUtilsTest.java

```diff
@@ -90,6 +98,28 @@ public void config() throws Exception {
 
         dataSourceRtMock = mock(DataSourceRT.class);
 
+        DataSource mockDataSource = mock(DataSource.class);
+        Connection mockConnection = mock(Connection.class);
+        DatabaseMetaData mockMetaData = mock(DatabaseMetaData.class);
+
+        when(mockDataSource.getConnection()).thenReturn(mockConnection);
+        when(mockConnection.getMetaData()).thenReturn(mockMetaData);
+        when(mockMetaData.getDatabaseProductName()).thenReturn("PostgreSQL");
+
+        mockStatic(ApplicationBeans.class);
+        when(ApplicationBeans.getBean(eq("databaseSource"), eq(DataSource.class)))
+                .thenReturn(mockDataSource);
+
+        java.lang.reflect.Field instanceField = DAO.class.getDeclaredField("instance");
+        instanceField.setAccessible(true);
+        instanceField.set(null, null);
+
+        PointValueDAO pointValueDAOMock = mock(PointValueDAO.class);
+        when(pointValueDAOMock.getLatestPointValue(anyInt())).thenReturn(System.currentTimeMillis());
+
+        mockStatic(PointValueDAO.class);
+        when(PointValueDAO.getInstance()).thenReturn(pointValueDAOMock);
```

Comment:
If you are not adding new test scenarios there should be no need to mock new classes.
Unit tests do not touch the database, they should be independent of the database used.
If there is a problem with unit testing, I will be happy to help, but if you use the template I suggested with the factor class, such changes in tests should not be necessary.